zio-http/src/main/scala/zio/http/Response.scala

```diff
@@ -331,9 +330,9 @@ object Response {
    * Creates a new response for the provided socket
    */
   def fromSocket[R](
-    http: Handler[R, Throwable, ChannelEvent[WebSocketFrame, WebSocketFrame], Unit],
+    http: Handler[R, Throwable, WebSocketChannel, Unit],
```

Comment:
We can allow any success value here:

```suggestion
    http: Handler[R, Throwable, WebSocketChannel, Any],
```